release_notes/v2.0.0.txt

```diff
@@ -118,6 +118,10 @@ Raft, and existing users should migrate to the newer Raft consensus type.  For
 compatibility with existing deployments, Kafka is still supported, but may be
 removed entirely in a future release.
 
+FAB-16477 and FAB-17116 The orderer config `general.genesismethod` and
```

Comment:
It would be nice to note that the old config values will still work, so long as the new ones are not set.  But that they should be considered to be deprecated.

Reply:
I think we can deprecate the old setting in v2.0?